spec/models/web_results_post_processor_spec.rb

```diff
@@ -81,9 +81,7 @@
         @post_processed_results = post_processor.post_processed_results
       end
 
-      # Temporarily disabling these specs during ES56 upgrade
-      # https://cm-jira.usa.gov/browse/SRCH-818
-      xit "should replace Bing's result title with the NewsItem title" do
+      it "replaces Bing's result title with the NewsItem title" do
         expect(@post_processed_results.first['title']).to eq("Title w/o highlighting")
```

Comment:
Let's use single quotes here.

Reply:
These need to be double quotes due to the apostrophe in `Bing's`.